purchase_request_to_requisition/models/product.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from openerp import api, models
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.multi
+    @api.constrains('purchase_request', 'purchase_requisition')
+    def _check_request_requisition(self):
+        for product in self:
+            if product.purchase_request and product.purchase_requisition:
```

Comment:
After reflexion, ```product.purchase_requisition``` is a radio button selection in version 9.0. Then, the second part of your condition will always be true. So, it's not possible to select ```purchase_request```